src/components/Headshots/Headshots.jsx

```diff
@@ -0,0 +1,270 @@
+			<h1>Meet the team</h1>
+			<div className="org">
+				<Headshot
+					name="Disala"
+					team="Co-Director"
+					role=" "
+					insta="disala_desilva"
+					linkedin="disaladesilva"
+					website=""
+				/>
+				<Headshot name="Mingye" team="Co-Director" role=" " insta="" linkedin="mingye-chen-ca" website="" />
+				<Headshot
+					name="Steven"
+					team="Community"
+					role="Director"
+					insta="_stevenli"
+					linkedin="-steven"
+					website=""
+				/>
+				<Headshot
+					name="Fatimah"
+					team="Design"
+					role="Director"
+					insta="fatimbit_v"
+					linkedin="fatimah-vakily"
+					website=""
+				/>
+				<Headshot
+					name="Daniel"
+					team="Development"
+					role="Director"
+					insta="arcanistzed"
+					linkedin="arcanistzed"
+					website=""
+				/>
+				<Headshot
+					name="Ali"
+					team="Finance"
+					role="Co-Director"
+					insta="notalibhangu"
+					linkedin="ali-raza-bhangu-061274214"
+					website=""
+				/>
+				<Headshot
+					name="Pavly"
+					team="Logistics"
+					role="Co-Director"
+					insta="pav_saleh"
+					linkedin="pavly"
+					website=""
+				/>
+				<Headshot
+					name="Tomer"
+					team="Logistics"
+					role="Co-Director"
+					insta="tomersz12"
+					linkedin="tomersz12"
+					website=""
+				/>
+				<Headshot
+					name="Nyah"
+					team="Marketing"
+					role="Director"
+					insta=""
+					linkedin="nyah-wagner-a62064192"
+					website=""
+				/>
+				<Headshot
+					name="Kian"
+					team="Sponsorship"
+					role="Director"
+					insta="kiiankiing"
+					linkedin="kiansalehi"
+					website=""
+				/>
+				<Headshot name="Brandon" team="Community" role="Coordinator" insta="" linkedin="" website="" />
+				<Headshot
+					name="Manaal"
+					team="Community"
+					role="Coordinator"
+					insta="manaalm20"
+					linkedin="manaal-mujeebuddin-ab2379199"
+					website=""
+				/>
+				<Headshot name="Rafael" team="Community" role="Coordinator" insta="" linkedin="rafaelarif" website="" />
+				<Headshot
+					name="Saheen"
+					team="Community"
+					role="Coordinator"
+					insta=""
+					linkedin="saheen-jeyarajah"
+					website=""
+				/>
+				<Headshot
+					name="Emma"
+					team="Design"
+					role="Coordinator"
+					insta="et.tangg"
+					linkedin="emmatang-"
+					website=""
+				/>
+				<Headshot name="Sakeena" team="Design" role="Coordinator" insta="" linkedin="" website="" />
+				<Headshot name="Sheena" team="Design" role="Coordinator" insta="" linkedin="sheenaverana" website="" />
+				<Headshot
+					name="Arjuunan"
+					team="Development"
+					role="Coordinator"
+					insta="itsarjuunan"
+					linkedin="arjuunan"
+					website=""
+				/>
+				<Headshot
+					name="Aydin"
+					team="Development"
+					role="Coordinator"
+					insta="ayay"
+					linkedin="aydinyalcinkaya"
+					website=""
+				/>
+				<Headshot
+					name="Emilien"
+					team="Development"
+					role="Coordinator"
+					insta=""
+					linkedin=""
+					website="https://edgie.emilien.ca/ "
```

Comment:
As I said earlier, the plan was *not* to put websites for anybody. If we're putting a website here, everyone should have the opportunity to put their website. I would remove the code for websites for now because asking everyone for that info is more of a hassle than it's worth.

Reply:
Still needs to be removed